examples/linear_model/plot_tweedie_regression_insurance_claims.py

```diff
@@ -0,0 +1,594 @@
+"""
+======================================
+Tweedie regression on insurance claims
+======================================
+
+This example illustrates the use of Poisson, Gamma and Tweedie regression on
+the French Motor Third-Party Liability Claims dataset, and is inspired by an R
+tutorial [1]_.
+
+Insurance claims data consist of the number of claims and the total claim
+amount. Often, the final goal is to predict the expected value, i.e. the mean,
```

Comment:
It'd be useful to briefly describe the dataset in more details here: what do the samples represent, and what are some of the features? Same for the other example.

Reply:
Mentioned a few relevant features, and added a link to the OpenML dataset which has the full list and as well as some feature statistics.